PictureApp.API/Startup.cs

```diff
@@ -88,7 +90,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                     });
                 });
             }
-
+            //seeder.SeedUsers();
```

Comment:
Why this line is commented?

Reply:
if you need generate the more data by 20181119_UserSeedData.json just uncomment this line and run your api. Its just only for development time

Reply:
My question was a little bit provocative ;) Are we able to use different approach instead of commented production code?

Reply:
I removed the seed data from startup.cs file, if you would like to generate temp data from json you need to prepare startap.cs class as fitst. commit:
https://github.com/anowakowski/PictureApp_API/pull/26/commits/64b0406d0c66f72c8dbe0f44b13213efd384b5b9